crates/re_viewer/src/ui/view_tensor/ui.rs

```diff
@@ -44,6 +44,15 @@ pub struct ViewTensorState {
     tensor: Option<DecodedTensor>,
 }
 
+// Manually implement PartialEq so we aren't comparing the tensor
```

Comment:
This is the third time now we manually skip things in `PartialEq`. Perhaps we should make them all link to the same github issue so we can find all instances quickly if/when we find a fix for it